src/driver/sap/SapQueryRunner.ts

```
@@ -237,8 +249,8 @@ export class SapQueryRunner extends BaseQueryRunner implements QueryRunner {
                });
            } catch (err) {
                 fail(err);
-            }
-        });
+           }
+        }).then(dropStatement, dropStatement)
```

Wait, this will swallow up any exceptions, won't it..?